.travis.yml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2018 Xaptum, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License
+
+language: c
+
+compiler: gcc
+
+env:
+  global:
+    - INSTALL_PREFIX=${TRAVIS_BUILD_DIR}/usr
+    - CMAKE_PREFIX_PATH=${INSTALL_PREFIX}
+    - LIBSODIUM_TAG=1.0.11
+    - LIBSODIUM_DIR=${TRAVIS_BUILD_DIR}/libsodium
+    - XTT_TAG=v0.10.1
+    - XTT_DIR=${TRAVIS_BUILD_DIR}/xtt/
+    - LIBCONFIG_TAG=1.7.2
+    - LIBCONFIG_DIR=${TRAVIS_BUILD_DIR}/libconfig
+    - LIBUV_TAG=v1.9.1
+    - LIBUV_DIR=${TRAVIS_BUILD_DIR}/libuv
+    - OPENSSL_TAG=1.1.1a
+    - OPENSSL_DIR=${TRAVIS_BUILD_DIR}/openssl
+    - USE_XTT=OFF
+
+before_script:
+  - ls
+  - pushd .travis/
+  - ls
+  - chmod +x *
+  - ./install-libconfig.sh ${LIBCONFIG_TAG} ${LIBCONFIG_DIR}
```

Comment:
Can we install these dependencies from [Bintray](https://bintray.com/xaptum/deb), rather than building from source?  That way these `enftun` builds are also testing that the required dependencies were properly published to our Bintray APT repo.